app/views/index/global.phtml

```diff
@@ -8,49 +8,50 @@
 		$class = ' state_unread';
 	}
 ?>
+<main>
+    <div id="stream" class="global<?= $class ?>">
```

Comment:
Shouldn't that be simply be main then?

Reply:
I did want to merge this tags, because the extension "ThreePanesView" looks like that it needs id=stream to make some magic (= add a column). I want to make sure that also this extension has a correct <main> wrapper

Reply:
I mean `<main id="stream" etc.>` of course, not `<main>`.

Reply:
I leave `<main>` and `<div id="stream">` splited, because the extension "ThreePanesView" searchs for id="stream"
![image](https://user-images.githubusercontent.com/1645099/120376848-a59d9280-c31c-11eb-894a-cb0819a53264.png)
and put its wrapper div around
![image](https://user-images.githubusercontent.com/1645099/120376920-bf3eda00-c31c-11eb-8598-9364be26076c.png)
In this way the <main> will wrap around "stream" and "threepanesview"

I would not change it.

Reply:
Could you please be more specific as to what the problem is? I'll fix the extension if necessary but I don't see it.

Reply:
Which information do you need?

Reply:
What is it that breaks?

Reply:
If `<main id="stream">` than, after ThreePanesView inserted its `<div id="ThreePanesView">`,  `<div id="ThreePanesView">` would not be IN the `<main>`